recipe/userroles/userrolesclaims/claims.go

```diff
@@ -0,0 +1,17 @@
+package userrolesclaims
+
+import "github.com/supertokens/supertokens-golang/recipe/session/claims"
+
+type TypeUserRoleClaimValidators struct {
+	claims.PrimitiveArrayClaimValidators
+}
```

Comment:
why do you need this type at all since there is nothing new in it?

Reply:
There is no special need, returning PrimitiveArrayClaimValidators instead